README.md

```diff
@@ -67,14 +69,16 @@ Read-only tokens are sufficient.
 | digitalocean_key                            | gauge   | 1            | Information about keys in your digitalocean account
 | digitalocean_loadbalancer_droplets          | gauge   | 1            | The number of droplets this load balancer is proxying to
 | digitalocean_loadbalancer_status            | gauge   | 1            | The status of the load balancer, 1 if active
+| digitalocean_month_to_date_balance          | gauge   | 1            | Balance as of the `digitalocean_balance_generated_at` time
+| digitalocean_month_to_date_usage            | gauge   | 1            | Amount used in the current billing period as of the `digitalocean_balance_generated_at` time
 | digitalocean_snapshot_min_disk_size_bytes   | gauge   | 2            | Minimum disk size for a droplet/volume to run this snapshot on in bytes
 | digitalocean_snapshot_size_bytes            | gauge   | 2            | Snapshot's size in bytes
 | digitalocean_start_time                     | gauge   | 1            | Unix timestamp of the start time
 | digitalocean_volume_size_bytes              | gauge   | 11           | Volume's size in bytes
 
 ### Alerts & Recording Rules
 
-As example alerts and recording rules I have copied my `.rules` file to this repository.  
+As example alerts and recording rules I have copied my `.rules` file to this repository.
```

Comment:
This has been an intentional new line break with the 2 spaces at the end.

Reply:
i'll fix those, my editor auto trims whitespace